modules/redis/src/testFixtures/java/com/loopers/testcontainers/RedisTestContainersConfig.java

```diff
@@ -0,0 +1,22 @@
+package com.loopers.testcontainers;
+
+import com.redis.testcontainers.RedisContainer;
+import org.springframework.context.annotation.Configuration;
+import org.testcontainers.utility.DockerImageName;
+
+@Configuration
+public class RedisTestContainersConfig {
+    private static final RedisContainer redisContainer = new RedisContainer(DockerImageName.parse("redis:latest"));
+
+    static {
+        redisContainer.start();
+    }
+
+    public RedisTestContainersConfig() {
+        System.setProperty("datasource.redis.database", "0");
+        System.setProperty("datasource.redis.master.host", redisContainer.getHost());
+        System.setProperty("datasource.redis.host.port", String.valueOf(redisContainer.getFirstMappedPort()));
+        System.setProperty("datasource.redis.replicas[0].host", redisContainer.getHost());
+        System.setProperty("datasource.redis.replicas[0].port", String.valueOf(redisContainer.getFirstMappedPort()));
+    }
+}
```

Comment:
Redis 테스트 컨테이너 Connection refused 오류납니다! 수정 필요해보입니다!

```suggestion
@Configuration
public class RedisTestContainersConfig {
    private static final RedisContainer redisContainer = new RedisContainer(DockerImageName.parse("redis:latest"));

    static {
        redisContainer.start();

        System.setProperty("datasource.redis.database", "0");
        System.setProperty("datasource.redis.master.host", redisContainer.getHost());
        System.setProperty("datasource.redis.master.port", String.valueOf(redisContainer.getFirstMappedPort()));
        System.setProperty("datasource.redis.replicas[0].host", redisContainer.getHost());
        System.setProperty("datasource.redis.replicas[0].port", String.valueOf(redisContainer.getFirstMappedPort()));
    }
}
```